python/semantic_kernel/planners/sequential_planner/sequential_planner.py

```diff
@@ -106,19 +106,17 @@ async def create_plan(self, goal: str) -> Plan:
             raise PlannerCreatePlanError(
                 f"Error creating plan for goal: {plan_result.metadata['error']}",
                 plan_result.metadata["error"],
-            )
+            ) from plan_result.metadata["error"]
```

Comment:
This is `exception` now everywhere else in the code, instead of `error`. Should it be `exception` here? And perhaps we should use a constant throughout our code base, so the value can be set/changed in one spot?